eq_translations/survey_schema.py

```diff
@@ -120,9 +120,10 @@ def get_list_pointers(self):
         for list_pointer in list_pointers:
             schema_element = resolve_pointer(self.schema, list_pointer)
             if isinstance(schema_element, list):
-                pointers.extend(
-                    [f"{list_pointer}/{i}" for i, p in enumerate(schema_element)]
-                )
+                for i, p in enumerate(schema_element):
+                    if not is_placeholder(p):
```

Comment:
I think it would be worth adding a comment here to say that placeholders are being skipped as they are dealt with elsewhere.

Reply:
👍